pkg/spec/backup.go

```diff
@@ -100,6 +101,19 @@ type S3Source struct {
 	AWSSecret string `json:"awsSecret,omitempty"`
 }
 
+// ABSSource represents an Azure Blob Storage (ABS) backup storage source
+type ABSSource struct {
+	// The name of the ABS container to store backups in.
```

Comment:
Go idiomatic comment starts with the variable name itself.

`ABSContainer is the name of the ABS container to store backups in.`?